molecule/default/converge.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    candlepin_user: "root"
+    candlepin_user_home: "/root/"
+    cp_configure_postgresql: true
+    cp_configure_mariadb: true
+    cp_configure_user_env: true
+    cp_configure_debugging: true
+    cp_git_checkout: true
+    cp_deploy: false
```

Comment:
Change this to "true" so we can have validations to verify CP starts up properly in the environment(s).